cmd/build/binary.go

```diff
@@ -42,8 +42,9 @@ func compile(ctx context.Context, cmd string, goos, goarch string) error {
 		"-X", fmt.Sprintf("'package-operator.run/internal/version.version=%s'", appVersion),
 	}
 
+	cmdPath := dotFilepathJoin("cmd", cmd)
 	err = shr.New(env).Run(
-		"go", "build", "--ldflags", strings.Join(ldflags, " "), "--trimpath", "--mod=readonly", "-o", dst, "./cmd/"+cmd,
+		"go", "build", "--ldflags", strings.Join(ldflags, " "), "--trimpath", "--mod=readonly", "-o", dst, cmdPath,
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed